docs/extensions/logging-and-monitoring.md

```diff
@@ -90,31 +97,34 @@ data:
 
 ## Logging
 
-The central fluentd/fluent-bit instances in the `garden` namespace are parsing the logs from all containers in the seed cluster.
-The shoot-specific instances only extract exactly those logs for the pods of the control plane they are responsible for.
-This allows to only fetch the logs for the pods once for the whole cluster, and to distribute them afterwards.
+In Kubernetes clusters, container logs are non-persistent and do not survive stopped and destroyed containers. Gardener addresses this problem for the components that it manages, by introducing its own managed logging solution. It is integrated with the Gardener monitoring stack to have all troubleshooting context in one place.
```

Comment:
```suggestion
In Kubernetes clusters, container logs are non-persistent and do not survive stopped and destroyed containers. Gardener addresses this problem for the components hosted in a seed cluster, by introducing its own managed logging solution. It is integrated with the Gardener monitoring stack to have all troubleshooting context in one place.
```

Gardener also manages some in-cluster components like kube-proxy and coredns, but no logs are stored for them.